Source/Views/TableViews/SPTableView.m

```diff
@@ -294,6 +295,28 @@ - (void)rightMouseDown:(NSEvent *)event
 	}
 }
 
+- (void)mouseDown:(NSEvent *)event
+{
+    if (event.modifierFlags & NSEventModifierFlagOption && event.modifierFlags & NSEventModifierFlagCommand){
+        if ([[[[self delegate] class] description] isEqualToString:@"SPTablesList"]) {
+            if ([[self delegate] respondsToSelector:@selector(openTableInNewTab:withRowIndex:)]) {
+                if ([[self delegate] respondsToSelector:@selector(tableView:shouldSelectRow:)]) {
+                    if ([[self delegate] tableView:self shouldSelectRow:[self rowAtPoint:[self convertPoint:[event locationInWindow] fromView:nil]]]) {
```

Comment:
This is a bunch of nested ifs. See the comment above about the early return paradigm. I for one really can't read this block of code to decipher what's happening

Reply:
![image](https://user-images.githubusercontent.com/7940795/153751858-02ecd06c-82a4-4d26-b145-94fefc7487c0.png)

Referring to the code in the `rightmousedown` method, I added the judgment of `sptableslist` and `opentableinnewtab: withrowindex:`.
As I said before, I'm not good at IOS development. I just make a simple implementation of my ideas.
I can't do better here. It's more difficult for me.
If you have a better way, you can also realize it separately, which is my honor.